CHANGELOG.md

```diff
@@ -1,7 +1,9 @@
 # OHHTTPStubs — CHANGELOG
 
 ## Master
-
+* Fixed potential memory leaks with use of NSURLSession as detected by our devs.
```

Comment:
Please add two spaces after the `.` (like you can see for other entries).

(This is a Markdown-specific stuff that makes the text on the next line actually be rendered in a new line in same paragraph instead of being joined on the same line)